samples/subsys/fs/littlefs_blk/boards/nucleo_h743zi.conf

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (c) 2021 Lukasz Majewski, DENX Software Engineering GmbH
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# SDMMC1 connected SD card (via only 3 wires; CLK, MOSI, MISO)
+CONFIG_DISK_ACCESS=y
+CONFIG_DISK_DRIVERS=y
+CONFIG_DISK_DRIVER_SDMMC=y
+
+CONFIG_DEBUG=y
+CONFIG_FILE_SYSTEM_LITTLEFS=y
+CONFIG_FLASH=y
+CONFIG_FLASH_MAP=y
+CONFIG_FLASH_PAGE_LAYOUT=y
+
+CONFIG_MINIMAL_LIBC_MALLOC_ARENA_SIZE=8192
+CONFIG_FS_LITTLEFS_FC_HEAP_SIZE=8192
```

Comment:
It it board specific?

Reply:
No, this is test specific and also shall be moved to prj.conf